nengo_sphinx_theme/theme/layout.html

```diff
@@ -69,6 +69,7 @@
         <div class="container">
           <div class="row">
             <div class="col-10 offset-1 pb-5 documentation-source" role="main">
+              {%- include "unstable_warning.html" %}
```

Comment:
Rather than having this included every time, can we do the check here and only include when we need to?

Reply:
> Rather than having this included every time, can we do the check here and only include when we need to?
Thats a good point, I have made the changes to the branch.